src/cross_section.F90

```diff
@@ -48,6 +51,10 @@ subroutine calculate_xs(p)
 
     mat => materials(p % material)
 
+    ! Find energy index on global or material unionized grid
+    if (grid_method == GRID_MAT_UNION) &
+      & call find_energy_index(p % E, p % material)
```

Comment:
The ampersand on the second line is superfluous (only needed if you are continuing a string). The style guide says continuation lines are indented with 5 extra spaces.